packages/runtime-core/src/apiWatch.ts

```diff
@@ -329,11 +329,11 @@ function doWatch(
         callWithAsyncErrorHandling(cb, instance, ErrorCodes.WATCH_CALLBACK, [
           newValue,
           // pass undefined as the old value when it's changed for the first time
-          oldValue === INITIAL_WATCHER_VALUE 
+          oldValue === INITIAL_WATCHER_VALUE
             ? undefined
-            : (isMultiSource && oldValue[0] === INITIAL_WATCHER_VALUE)
-              ? []
-              : oldValue,
+            : isMultiSource && oldValue[0] === INITIAL_WATCHER_VALUE
+            ? []
+            : oldValue,
```

Comment:
This part of the code change is probably not necessary because it has nothing to do with the problem I'm trying to solve. It is automatically formatted on commit and I don't know how to avoid this process.

Reply:
It's okay. Just keep it.